cmd/kubeadm/app/apis/kubeadm/v1beta1/defaults_windows.go

```diff
@@ -22,5 +22,5 @@ const (
 	// DefaultCACertPath defines default location of CA certificate on Windows
 	DefaultCACertPath = "C:/etc/kubernetes/pki/ca.crt"
 	// DefaultUrlScheme defines default socket url prefix
-	DefaultUrlScheme = "tcp"
+	DefaultUrlScheme = "npipe"
```

Comment:
Strictly speaking we shouldn't be doing this, but since we don't have a released working version of kubeadm on Windows, that has support for v1beta1, I don't think that someone uses it. Hence, in my view, it's OK to be changed.